components/hab/src/cli.rs

```diff
@@ -143,12 +143,15 @@ mod tests {
 
     #[test]
     fn legacy_appliction_and_environment_args() {
-        let r = get(no_feature_flags()).get_matches_from_safe(vec!["hab",
-                                                                   "sup",
-                                                                   "run",
-                                                                   "--application",
-                                                                   "--environment=env"]);
-        assert!(r.is_ok());
+        #[cfg(not(target_os = "macos"))]
```

Comment:
I don't think we want to add this constraint. None of our pipelines run these tests on macos. If we do add macos, then we would likely want them to run these tests.

Reply:
Removed